app/views/partials/_add_payload.erb

```diff
@@ -0,0 +1,10 @@
+<div class="panel-body">
```

Comment:
i'd probably suggest naming these partials something different. like `_payload_form`. or even just having the form within a payload folder!